bundle/src/bundler.rs

```diff
@@ -65,11 +74,24 @@ impl BundlerUtil {
             })?;
 
         if let Some(CodeOwners { ref path, .. }) = self.meta.base_props.codeowners {
-            let mut file = std::fs::File::open(path)?;
+            let mut file = File::open(path)?;
             tar.append_file("CODEOWNERS", &mut file)?;
             total_bytes_in += std::fs::metadata(path)?.len();
         }
 
+        if let Some(bep_parser) = self.bep_parser.as_ref() {
+            let mut bep_events_file = tempfile::tempfile()?;
+            bep_parser.bep_test_events().iter().for_each(|event| {
```

Comment:
This assumes that `bep_parser.parse()` was called somewhere upstream (and I do see that it was called in the upload flow before being passed to the `BundlerUtil` here). Curious if you've considered having the `BazelBepParser` output a single struct with all of the relevant parsed data that we can pass around rather than the parser itself?

Reply:
Agreed—might be better to have a struct which represents an already parsed state

Reply:
Good point. I'm refactoring to handle that